CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Refined oneDrive rate limiter controls to reduce throttling errors.
 - Fix handling of duplicate folders at the same hierarchy level in Exchange. Duplicate folders will be merged during restore operations.
 - Fix backup for mailboxes that has used up all their storage quota
+- Don't show restored folders in CLI restore output. Only show restored items instead.
```

Comment:
```suggestion
- Restored folders will no longer appear in the Restore results. Only items will get displayed.
```

Reply:
thanks! guess I should put more thought into my changelog messages 😅

Reply:
heh, naw.  entirely a nitpick; it would've been fine.